_gsocproposals/2020/proposal_CMS-Monitoring.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Intelligent Alert Management System For HEP experiments
+layout: gsoc_proposal
+project: CMS-Monitoring
+year: 2020
+organization: CERN
```

Comment:
You may need to add also the other organizations mentors are from (CERN, Cornell and INFN), see for 
example _gsocproposals/2019/proposal_TMVANNOptimisation.md